From: Matthew Wilcox <matthew.r.wilcox@intel.com>
Date: Mon, 7 Feb 2011 15:55:59 -0500
Subject: NVMe: Renumber the special context values
Git-commit: d2d8703481f60d67f49e3177196cbe474b11377c
References: FATE#313627
Patch-Mainline: v3.7-rc1

If POISON_POINTER_DELTA isn't defined, ensure they're in page 0 which
should never be mapped.

Signed-off-by: Matthew Wilcox <matthew.r.wilcox@intel.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/block/nvme.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/block/nvme.c b/drivers/block/nvme.c
index 2a0dd5e..71bdf6f 100644
--- a/drivers/block/nvme.c
+++ b/drivers/block/nvme.c
@@ -184,9 +184,9 @@ enum {
 };
 
 #define CMD_CTX_BASE		(POISON_POINTER_DELTA + sync_completion_id)
-#define CMD_CTX_CANCELLED	(0x2008 + CMD_CTX_BASE)
-#define CMD_CTX_COMPLETED	(0x2010 + CMD_CTX_BASE)
-#define CMD_CTX_INVALID		(0x2014 + CMD_CTX_BASE)
+#define CMD_CTX_CANCELLED	(0x30C + CMD_CTX_BASE)
+#define CMD_CTX_COMPLETED	(0x310 + CMD_CTX_BASE)
+#define CMD_CTX_INVALID		(0x314 + CMD_CTX_BASE)
 
 static unsigned long free_cmdid(struct nvme_queue *nvmeq, int cmdid)
 {
-- 
1.7.4.2

